sphinx_rtd_theme/layout.html

```diff
@@ -112,6 +112,11 @@
   {%- endif %}
   {%- endblock %}
   {%- block extrahead %} {% endblock %}
+
+  <meta name="twitter:card" content="summary"/>
+  <meta name="twitter:site" content="@ciliumproject"/>
+  <meta name="twitter:title" content="{{ title|striptags|e }}{{ titlesuffix }}"/>
+  <meta name="twitter:image" content="{{ canonical_url }}{{ pathto('_static/' + logo, 1) }}"/>
```

Comment:
@genbit Is this the proper way to get the full URL to the docs? I see that e.g. on line 175 we hardcode the URL, so maybe I should just do that.

Reply:
@pchaigno I'm not sure actually. We can try deploying it on readthedocs and see if it works.

Reply:
Test in production? One new thing off my bucket list :ballot_box_with_check:

Reply:
@genbit Should we merge this then?

Reply:
done